typescript/react/security/audit/react-no-refs.yaml

```diff
@@ -7,13 +7,15 @@ rules:
   - pattern: createRef(...)
   - pattern: useRef(...)
   message: >-
-    `ref` usage found, refs give direct DOM access and may create a possibility for XSS
+    `ref` usage found, refs give direct DOM access and may create a possibility for XSS, which could cause account compromise. Instead, avoid direct DOM manipulation or use DOMPurify to sanitize HTML before writing it into the page.
```

Comment:
How does this cause account compromise?

Reply:
I was thinking xss could lead to cookie compromise, which could possibly lead to session takeover (and wrote account compromise as a broader category of session takeover 😅 ). I'll say `which could cause sensitive information such as user cookies to be exfiltrated by an attacker` instead 👍